domains/www.viggu.json

```
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "VIGGU-7",
+        "email": "vigneshpatlavath@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "viggu-7.github.io"
```

Perhaps you meant your portfolio website?
```suggestion
        "URL": "https://viggu-7.github.io/portfolio"
```